### ISSUE TITLE
Cannot create merge request

### ISSUE THREAD

**Issue by michalhudecek** [PavlikPolivka/GitLabProjects#69]
Windows 10
PhpStorm 2017.2

When I try to Create merge request I always get this error:
`Cannot Create Merge Request: Cannot find this project in GitLab Remote`

I can however list all the projects from GitLab when I go to VCS > Checkout from version control > Git > GitLab I do see the current project in the list.

I think this started to happen after previous update (one before today one).

**Comment by PavlikPolivka** [comment]
it tries to guess the gitlab project from remote url ... to debug this kind of issues itwould be easier if you can type what is the url for gitlab api in the setting dialog and what is the remote of the repo (git remote -v), you can obsure it somehow if you want, but it would be much more easier to figure this out with this kind of info

thanks

**Comment by michalhudecek** [comment]
The URL in settings is https://gitlab.com
The URL of the repo is in this format https://gitlab.com/SomeName/SomeName
First SomeName = group, second SomeName = name of the repo

**Comment by PavlikPolivka** [comment]
Strange, I have a unit test for this and these combinations should be working.

There is no .git at the end of the repo url?

**Comment by michalhudecek** [comment]
It actually is via SSH. I checked git config and I think the problem might be in lowercase/uppercase. 

The full SSH address on GitLab site is `git@gitlab.com:SomeName/SomeName.git`
In git config I have: `git@gitlab.com:SomeName/somename.git`

**Comment by michalhudecek** [comment]
I can confirm the hypothesis above - it started working after I changed the address in git config to git@gitlab.com:SomeName/SomeName.git.

However, I haven't added the address manually. I cloned the repo from VCS > Checkout from version control > GitLab. Maybe the origin of this issue is there?

**Comment by PavlikPolivka** [comment]
Yes. You are right. Good catch.

I do not have a power over what url is used during checkout, I just point IDE git integration over to the gitlab project. 
I fixed the class that is doing to matching to always match everything in lower case. 
From what I can tell with my testing it works when I change case in the remote url.

I released this fix.
This most likely also fixes issues #65 #55 an #49
